tests/modules/fastp/main.nf

```diff
@@ -11,8 +11,9 @@ workflow test_fastp_single_end {
     input = [ [ id:'test', single_end:true ], // meta map
               [ file(params.test_data['sarscov2']['illumina']['test_1_fastq_gz'], checkIfExists: true) ]
             ]
+    save_trimmed_fail = Channel.value(false)
```

Comment:
Didn't know this one! :love_you_gesture:

Reply:
`save_trimmed_fail = false` is fine :)

Reply:
Even better :heart_eyes: